airflow/www/package.json

```diff
@@ -28,51 +28,51 @@
   "devDependencies": {
     "babel": "^6.23.0",
     "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.4",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
     "babel-plugin-css-modules-transform": "^1.6.1",
     "babel-polyfill": "^6.26.0",
-    "clean-webpack-plugin": "^0.1.19",
-    "copy-webpack-plugin": "^5.1.1",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^6.0.3",
     "css-loader": "^3.4.2",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb-base": "^13.0.0",
-    "eslint-plugin-html": "^4.0.5",
+    "eslint": "^7.5.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-plugin-html": "^6.0.2",
     "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-node": "^7.0.1",
-    "eslint-plugin-promise": "^3.8.0",
-    "eslint-plugin-standard": "^3.1.0",
-    "file-loader": "^1.1.11",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "file-loader": "^6.0.0",
     "handlebars": "^4.5.3",
-    "imports-loader": "^0.8.0",
-    "jquery": ">=3.4.0",
+    "imports-loader": "^1.1.0",
+    "jquery": "^3.5.1",
```

Comment:
jquery 3.5.0 previously broke the `pause/unpause` functionality. Have you tested the webserver after the changes in the PR @retornam ?

Reply:
Details: https://github.com/dpgaspar/Flask-AppBuilder/issues/1362 | https://github.com/apache/airflow/issues/8613 | https://github.com/apache/airflow/issues/8599

Reply:
@kaxil I started the webserver, logged in but didn't test the example dags. I'll run more detailed testing and fix bugs I find.

Reply:
@retornam while you are doing the tests, could you document and share with us on what UI functionalities you have tested? That way we can also double check on it with different data or see if any major feature is being missed.

Reply:
Full details of what I did. I ended up upgrading each package separately. We cannot upgrade the following packages
- d3
- nvd3 (original package is no longer maintained and there are two active forks)
- jquery due to a bug with CSRF support in Flask App Builder https://github.com/dpgaspar/Flask-AppBuilder/issues/1362
- moment ( upgrading to the new  version broke moment support completely)


For UI testing, I checked the following pages 
- /home 
    -  made sure there were no new JS errors / CSS errors in the console.
    -  turned the example dags on and off 
    - triggered dags button 
    - clicked on tree view button
    - clicked on graph view button
    - clicked on task view button
    - clicked on task tries button
    - clicked on landing times button
    - clicked on the code button
    - clicked on the refresh button
    - clicked on the delete dag button
    - performed a search for  a dag using the search button
    - filtered dags using the string example
    - reset the filtered dags
    - clicked on last run for the example_bash operator which I enabled
    - clicked on the successful, failed and running dag links
    - clicked on the recent tasks icon
    - switched tabs between all, active and paused
 - /users/list/
   - added a new user
   - deleted a new user
-/tree for a dag
   - verified it displays the tree
-/graph for a dag 
   - verified the graph renders
-/duration for a dag
  - verified  the graph renders
-/tries for a dag
 - verified graph renders
-/landing_times for a graph
 - verified graph renders
-/gant for a dag
  - verified graph renders
-/dag_details for a dag
 - verified page logs
-/code for a dag
  - verified page loads
- verified the trigger dag, refresh dag and delete dag buttons work on all pages
- logged out
- logged in

I tried to cover as many pages that include  the scripts `moment.*.js`, `base.*.js` and `d3.min.js`

Reply:
@houqp  have you been able to test this branch?

Reply:
> jquery due to a bug with CSRF support in Flask App Builder dpgaspar/Flask-AppBuilder#1362

This hasn't been fixed? @kaxil

Reply:
@kaxil you answered that it was fixed in 2.3.4?! it isn't?

Reply:
I think `d3` and `nvd3` is not worth the time we need to spend to upgrade the UI.

If we build a new UI from scratch then we should keep that in mind or use a different library then d3.

Reply:
> > jquery due to a bug with CSRF support in Flask App Builder [dpgaspar/Flask-AppBuilder#1362](https://github.com/dpgaspar/Flask-AppBuilder/issues/1362)
> 
> @kaxil you answered that it was fixed in 2.3.4?! it isn't?

I believe "fixed" just means "FAB 2.3.4 [reverted](https://github.com/dpgaspar/Flask-AppBuilder/pull/1363) previous `jQuery` bump".  It's not a fix but just a workaround.

Reply:
Yeah that is correct, jQuery 3.5 still causes issue

Reply:
Ah yeah I am remembering now.